components/Cards/tests/unit/CardsMaterialThemingTests.swift

```diff
@@ -99,11 +99,10 @@ class CardsMaterialThemingTests: XCTestCase {
     }
 
     // Test remaining properties
-    XCTAssertEqual(card.borderWidth(for: .normal), 1, accuracy: 0.001)
-    XCTAssertEqual(card.borderWidth(for: .selected), 1, accuracy: 0.001)
-    XCTAssertEqual(card.borderWidth(for: .highlighted), 1, accuracy: 0.001)
-    XCTAssertEqual(card.borderWidth(for: .disabled), 1, accuracy: 0.001)
-    XCTAssertEqual(card.shadowElevation(for: .highlighted), ShadowElevation(rawValue: 4))
+    [UIControlState.normal, .highlighted, .selected, .disabled].forEach {
+      XCTAssertEqual(card.shadowElevation(for: $0), ShadowElevation.none)
+      XCTAssertEqual(card.borderWidth(for: $0), 1, accuracy: 0.001)
```

Comment:
Add the `$0` to the assertion message please; this will let us know which state failed in case one of these assertions fails.

Reply:
Done